packages/tailwindcss-language-service/src/completions/file-paths.test.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from 'vitest'
+import { findFileDirective } from './file-paths'
+
+let findV3 = (text: string) => findFileDirective({ enabled: true, v4: false }, text)
```

Comment:
This doesn't seem to be used. I'm assuming you want to add more tests for v3 and v4.

Reply:
Forgot to add the tests there. Added a bunch more. Good catch.